packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -247,6 +249,43 @@ export const initProvider = () => (dispatch: Dispatch) => {
     return decision.promise;
 };
 
+const getMetadataFilesList = () => async (dispatch: Dispatch) => {
+    const provider = dispatch(getProvider());
+    if (!provider) {
+        return;
+    }
+
+    const listResponse = await provider.getMetadataFilesList();
+    if (!listResponse.success) {
+        return;
+    }
+
+    // remove the extention
+    const trimmedNames = listResponse.payload?.map(name => name.slice(0, -5));
+
+    return trimmedNames;
+};
+
+const getMetadataFilesMinimumVersion = (filesList: string[] | undefined) => {
+    let versionNumber = METADATA_ENCODING_VERSION;
+    if (!filesList || !filesList.length) {
+        return versionNumber;
+    }
+
+    filesList.every(name => {
```

Comment:
although this works, it always catches my attention. `array.every` is used where `array.forEach` is intended to be used. 

I suggest doing either

```
const versionNumber = filesList.every(name => name.includes('_')) ? 2 : 1
```
or to 
```
  filesList.forEach(name => {
        // v1 files do not have a postfix
        if (!name.includes('_')) {
            versionNumber = 1;
        }
    });

```

Reply:
done